Stylesheets/authentication.css

```diff
@@ -0,0 +1,57 @@
+.auth-main-container {
+    min-width: 33rem;
+    margin: auto;
+    position: absolute;
+    left: 43rem;
+    top: 16rem;
+    padding: 2rem;
+    border-radius: 4px;
+}
+
+.auth-header h3 {
+    font-size: 2rem;
+    font-weight: 2000;
+    text-align: center;
+    margin-bottom: 2rem;
+    color: var(--st-primary-color);
+}
+
+.auth-input label{
+    font-style: normal;
+    font-weight: 1000;
+    font-size: 1.2rem;
```

Comment:
Same here for font-weight and if possible use variables for weights.

Reply:
sure thanks